glib/src/gstring.rs

```diff
@@ -404,6 +404,29 @@ impl<'a> ToGlibPtr<'a, *mut c_char> for GString {
     }
 }
 
+#[doc(hidden)]
+impl<'a> FromGlibContainer<*const c_char, *const i8> for GString {
```

Comment:
For completeness, can you also implement it for `*mut _` and `u8`?

Reply:
You can implement that all by calling into the other implementation by casting the raw pointers.

Reply:
> You can implement that all by calling into the other implementation by casting the raw pointers.

I was going to copy paste them :P